.github/README.md

```diff
@@ -1,16 +1,32 @@
-# UndertaleModTool
-*(seeing such an amazing tool...<br/>
-... fills you with DETERMINATION.)*
+<p align="center">
+  <img src="images/logo.png" alt="UndertaleModTool Logo"/>
+</p>
 
-Heya. I heard you like digging deep into Undertale data so I made a tool just for you! 
+![Discord](https://img.shields.io/discord/566861759210586112?label=Discord&logo=discord&logoColor=white) ![GitHub](https://img.shields.io/github/license/krzys-h/UndertaleModTool?logo=github)
 
-**Downloads** are **HERE**: :point_right::point_right::point_right: https://github.com/krzys-h/UndertaleModTool/releases :point_left::point_left::point_left:
+<p align="center">(seeing such an amazing tool it fills you with DETERMINATION.)</p>
+<p align="center">Heya. I heard you like digging deep into Undertale/Deltarune or any GameMaker game data so I made a tool just for you!</p>
+<p align="center">
+  <img src="images/flowey.gif" alt="Flowey"/>
+</p>
 
-**Have questions? Want to talk?** Join us on Discord! https://discord.gg/RxXpdwJ
+# Downloads
 
-![flowey](flowey.png)
+Looking for the most stable or the latest avaliable download? Both the latest stable release and the most cutting edge version are in the table below!
+
+| Releases 	| Status 	|
+|:---:	|----------	|
+| Stable[^1][^2] 	| ![GitHub release (latest by date)](https://img.shields.io/github/downloads/krzys-h/UndertaleModTool/latest/total) |
```

Comment:
Clicking the release badge does not link to the actual release page which would be devastating for new users.